src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/TestUriHelper.cs

```diff
@@ -36,6 +36,12 @@ public static int FindFreePort(int initialPort)
         {
```

Comment:
The only reason it existed in the first place was because of port conflict issues in IIS, so yeah, as long as we're switching IIS over to always using a dynamic port we can remove it.